common/static/common/js/discussion/views/new_post_view.js

```diff
@@ -161,8 +162,16 @@
                     },
                     error: DiscussionUtil.formErrorHandler(this.$('.post-errors')),
                     success: function(response) {
-                        var thread;
+                        var thread, discussionBoardModel;
```

Comment:
It will be more clear if we rewrite `discussionBoardModel ` as `discussionBreadcrumbsModel`.